src/templates/macros.html

```diff
@@ -0,0 +1,59 @@
+{% macro summary_line(kind, covered, total) %}
+    {% set per = percent(num=covered, den=total) %}
+    <div class="level-item has-text-centered">
+        <div>
+            <p class="heading">{{ kind | capitalize }}</th>
```

Comment:
You must run a html validator against the different possible generated htmls to be sure that everything is fine.
For example it'd have helped you to get a spurious `</th>` here.

Reply:
I'll do.

Reply:
tidy-html5 seems happy now